codecov.yml

```diff
@@ -3,7 +3,7 @@ coverage:
     project:
       default:
         enabled: yes
-        target: auto
+        target: 85%
```

Comment:
I removed the test files from the coverage report, so the overall coverage has decreased.